invertedai/logs/debug_logger.py

```diff
@@ -0,0 +1,77 @@
+from pydantic import BaseModel, validate_arguments
+from typing import List, Optional, Dict, Tuple
+from datetime import datetime
+
+import json
+
+class DebugLogger:
+
+    def __init__(self,debug_log_path):
+        self.debug_log_path = debug_log_path
+
+        self.data = {
+            "location_info_requests":[],
+            "location_info_responses":[],
+            "location_info_request_timestamps":[],
+            "location_info_response_timestamps":[],
+            "initialize_requests":[],
+            "initialize_responses":[],
+            "initialize_request_timestamps":[],
+            "initialize_response_timestamps":[],
+            "drive_requests":[],
+            "drive_responses":[],
+            "drive_request_timestamps":[],
+            "drive_response_timestamps":[],
+        }
+
+        file_name = "iai_log_" + self._get_current_time_human_readable_UTC() + "_UTC.json"
+        self.log_path = self.debug_log_path + file_name
+
+    def _create_directory(self):
+        if not os.path.isdir(self.debug_log_path):
+            os.makedirs(self.debug_log_path)
```

Comment:
i think we should use logger.info and logger.warning here instead of print

Reply:
Will do

Reply:
Done